### PR TITLE
added PlayerNameEntityEvent

### PR DESCRIPTION
**Pull request by Machine-Maker**
Adds an event for when a player renames an entity

### CODE REVIEW

File: Spigot-API-Patches/0215-added-PlayerNameEntityEvent.patch [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+Subject: [PATCH] added PlayerNameEntityEvent
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/event/player/PlayerNameEntityEvent.java b/src/main/java/com/destroystokyo/paper/event/player/PlayerNameEntityEvent.java
+new file mode 100755
+index 0000000000000000000000000000000000000000..b095e3bd42f7ea68d0e30246649fe2b29745f9a0
+--- /dev/null
++++ b/src/main/java/com/destroystokyo/paper/event/player/PlayerNameEntityEvent.java
+@@ -0,0 +1,118 @@
++package com.destroystokyo.paper.event.player;
```
Comment (Proximyst) [inline Spigot-API-Patches/0215-added-PlayerNameEntityEvent.patch]:
```suggestion
+package io.papermc.paper.event.player;
```

**Review by Proximyst — DISMISSED** [review]

**Comment by MiniDigger** [comment]
what would be the usecase for overriding the entity?

**Comment by Machine-Maker** [comment]
I'm sure there's something. I mean that's mainly in there because it was easy to add. Probably some plugin author can come up with a way to use it. Force a player to rename one specific entity, or something like that.

**Comment by Machine-Maker** [comment]
Ok, this was rebased.

**Review by electronicboy — CHANGES_REQUESTED** [review]
a d v e n t u r e

**Comment by Machine-Maker** [comment]
Yeah, I have several PRs that are just waiting on adventure :)

**Comment by MiniDigger** [comment]
@Machine-Maker mind updating?

**Comment by Machine-Maker** [comment]
@MiniDigger done

**Pull request merged**